Guidelet/GuideletLib/UltraSound.py

```diff
@@ -4,12 +4,13 @@
 import time
 
 class UltraSound(object):
-
   DEFAULT_IMAGE_SIZE = [800, 600, 1]
 
+  # -----------------------------------------------------
```

Comment:
We don't add separator lines as text to py files, because IDE's can draw separator lines automatically (and having the separators as text would also make code folding uglier - instead of seeing only the methods, there would be always that manual separator line displayed as well). What Python IDE do you use?